### PR TITLE
Update Java Doc in Bigtable config, table and scan config.

### PR DESCRIPTION
**Pull request by kevinsi4508**

### CODE REVIEW

**Comment by sduskis** [comment]
Can you please ask someone else to review this?  It would be great to get more opinions.

**Comment by kevinsi4508** [comment]
I would prefer you review it first. thanks!

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java [hunk 1]
```diff
@@ -49,8 +50,8 @@ public static CloudBigtableScanConfiguration fromCBTOptions(CloudBigtableOptions
   }
 
   /**
-   * Converts a {@link CloudBigtableOptions} object to a {@link CloudBigtableScanConfiguration},
-   * with the addition of a {@link Scan} that filters the table.
+   * Converts a {@link CloudBigtableOptions} object to a {@link CloudBigtableScanConfiguration}
+   * object with the addition of a {@link Scan} that filters the table.
```
Comment (mgarolera) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: what about  "...CloudBigtableScanConfiguration that will perform the specified Scan on the table." ?

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java [hunk 2]
```diff
@@ -195,30 +196,30 @@ public boolean equals(Object obj) {
   private SerializableScan serializableScan;
 
   /**
-   * Creates a {@link CloudBigtableScanConfiguration} using the specified information.
+   * Creates a {@link CloudBigtableScanConfiguration} using the specified configuration.
```
Comment (mgarolera) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: "Creates a {@link CloudBigtableScanConfiguration} that will perform the specified scan in the table."
?

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java [hunk 3]
```diff
@@ -195,30 +196,30 @@ public boolean equals(Object obj) {
...
-      String table, Scan scan) {
-    this(projectId, zone, cluster, table, scan, Collections.<String, String> emptyMap());
+  public CloudBigtableScanConfiguration(String projectId, String zoneId, String clusterId,
+      String tableId, Scan scan) {
+    this(projectId, zoneId, clusterId, tableId, scan, Collections.<String, String> emptyMap());
   }
 
   /**
-   * Creates a {@link CloudBigtableScanConfiguration} using the specified information.
+   * Creates a {@link CloudBigtableScanConfiguration} using the specified configuration.
```
Comment (mgarolera) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: "Creates a {@link CloudBigtableScanConfiguration} that will perform the specified scan in the table, with additional configuration options."

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java (hunk 2 above)
Comment (kevinsi4508) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: We can't just mention scan, there are other parameters that are part of the configuration.
We either don't mention each and every one and let @param tag do that or mention all of them.

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java (hunk 1 above)
Comment (kevinsi4508) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: done

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java (hunk 3 above)
Comment (kevinsi4508) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: same as above, do we want mention all the config parameters here?

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java (hunk 2 above)
Comment (mgarolera) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: I think it is reasonable to describe together all the ID paramenters into "location" or "table", and separetely the scan, which acts on the location specified by the others.

Up to you.

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java (hunk 3 above)
Comment (mgarolera) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: I think it is reasonable to explain in the docs what is the difference in the input fields.

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java (hunk 2 above)
Comment (kevinsi4508) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: done

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java (hunk 3 above)
Comment (kevinsi4508) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: done.

**Comment by kevinsi4508** [comment]
Marcal is off to his vacation.

**Comment by sduskis** [comment]
LGTM

**Comment by kevinsi4508** [comment]
Jeff, please also take a look.

**Comment by hegemonic** [comment]
LGTM

**Pull request merged**